assets/airports/eddm.json

```diff
@@ -1,883 +1,2695 @@
+                ["ROKIL", "DM411", "DM412", "DM427", "DM426", "DM425", "DM424", "DM423", "DM422", "DM421", "DM420"]
+            ]
+        },
+        "ROKIL26": {
+            "icao": "ROKIL26",
+            "name": "Rokil Two Six",
+            "entryPoints": {
+                "ROKIL": ["ROKIL"]
+            },
+            "body": ["DM421", "DM422", "DM423", "DM424", ["DM425", "A80"], "DM426", "DM427", "DM428", "DM429"],
+            "draw": [
+                ["ROKIL", "DM421", "DM422", "DM423", "DM424", "DM425", "DM426", "DM427", "DM428", "DM429"]
+            ]
+        },
+        "LANDU08": {
+            "icao": "LANDU08",
+            "name": "Landu Zero Eight",
+            "entryPoints": {
+                "LANDU": ["LANDU"]
+            },
+            "body": ["DM428", "DM427", "DM426", "DM425", "DM424", ["DM423", "A80"], "DM422", "DM421", "DM420"],
+            "draw": [
+                ["LANDU", "DM428", "DM427", "DM426", "DM425", "DM424", "DM423", "DM422", "DM421", "DM420"]
+            ]
+        },
+        "LANDU26": {
+            "icao": "LANDU26",
+            "name": "Landu Two Six",
+            "entryPoints": {
+                "LANDU": ["LANDU"]
+            },
+            "body": ["DM412", "DM411", "DM422", "DM423", "DM424", ["DM425", "A80"], "DM426", "DM427", "DM428", "DM429"],
+            "draw": [
+                ["LANDU", "DM412", "DM411", "DM422", "DM423", "DM424", "DM426", "DM427", "DM428", "DM429"]
+            ]
+        },
+        "NAPSA08": {
+            "icao": "NAPSA08",
+            "name": "Napsa Zero Eight",
+            "entryPoints": {
+                "NAPSA": ["NAPSA"]
+            },
+            "body": ["DM458", "DM457", "DM456", "DM455", "DM454", ["DM453", "A80"], "DM452", "DM451", "DM450"],
+            "draw": [
+                ["NAPSA", "DM458", "DM457", "DM456", "DM455", "DM454", "DM453", "DM452", "DM451", "DM450"]
+            ]
+        },
+        "NAPSA26": {
+            "icao": "NAPSA26",
+            "name": "Napsa Two Six",
+            "entryPoints": {
+                "NAPSA": ["NAPSA"]
+            },
+            "body": ["DM467", "DM466", "DM465", "DM464", "DM463", "DM452", "DM453", "DM454", ["DM455", "A80"], "DM456", "DM457", "DM458", "DM459"],
+            "draw": [
+                ["NAPSA", "DM467", "DM466", "DM465", "DM464", "DM463", "DM452", "DM453", "DM454", "DM455", "DM456", "DM457", "DM458", "DM459"]
+            ]
+        },
+        "BETOS08": {
+            "icao": "BETOS08",
+            "name": "Betos Zero Eight",
+            "entryPoints": {
+                "BETOS": ["BETOS"]
+            },
+            "body": ["DM472", "DM473", "DM474", "DM475", "DM476", "DM457", "DM456", "DM455", "DM454", ["DM453", "A80"], "DM452", "DM451", "DM450"],
+            "draw": [
+                ["BETOS", "DM472", "DM473", "DM474", "DM475", "DM476", "DM457", "DM456", "DM455", "DM454", "DM453", "DM452", "DM451", "DM450"]
+            ]
+        },
+        "BETOS26": {
```

Comment:
Are these `BETOS08` and `BETOS26` procedures applicable by making a `BETOS` arrival with suffixes `"08": "08", "26": "26"`?

We should do as much as we can to simplify these files, both to reduce their size, and to increase their readability.